### DISCUSSION TITLE
Any samples on how to use .obj files with FXGL?

### DISCUSSION THREAD

**Discussion by ImperaEtConquer** [AlmasB/FXGL#1064] [Q&A]
Hey, everyone!

Want to try out new FXGL 11.16 features but I have no experience with 3D development.
Are there any samples on how to import Model3D from .obj and put it in the game?

Couldn't find anything here:
https://github.com/AlmasB/FXGL/tree/dev/fxgl-samples/src/main/java

Does FXGL also supports .mtl?

**Comment by AlmasB** [comment]
Thanks, [added a sample](https://github.com/AlmasB/FXGL/commit/d8cd019fe81e10da5f0b5871d6f64c1382ec3fce)

**Reply by AlmasB** [reply]
Yes .mtl is also supported to some extent (some materials may not be parsed, if this happens, feel free to provide an example, so the support can be improved).

**Reply by ImperaEtConquer** [reply]
Thank you!
